polaris-react/src/components/Popover/components/PopoverOverlay/PopoverOverlay.tsx

```diff
@@ -255,19 +241,17 @@ export class PopoverOverlay extends PureComponent<PopoverOverlayProps, State> {
           tabIndex={0}
           onFocus={this.handleFocusFirstItem}
         />
-        <UseFeatures>
-          {(features) => (
-            <Box
-              position="relative"
-              overflowX="hidden"
-              overflowY="hidden"
-              background="bg"
-              borderRadius={features.polarisSummerEditions2023 ? '3' : '2'}
-            >
-              {content}
-            </Box>
-          )}
-        </UseFeatures>
+        <div className={styles.ContentContainer}>
+          <div
+            id={id}
+            tabIndex={autofocusTarget === 'none' ? undefined : -1}
+            className={contentClassNames}
+            style={contentStyles}
+            ref={this.contentNode}
+          >
+            {renderPopoverContent(children, {captureOverscroll, sectioned})}
+          </div>
+        </div>
```

Comment:
☝️ Main change here is to replace `<Box [...props]>` with `<div className="...">`, then I could remove `<UseFeatures>` and inline `content`.